tests/test_schema_syntax.py

```diff
@@ -358,6 +359,8 @@ def test_eschema_syntax_type_19(self):
         };
         """
 
+    @test.xfail('not sure how to test multi-line semantics, '
```

Comment:
@vpetrovykh Any suggestions here?

Reply:
Why is it `xfailed`? This should be testable even with strings that forget their original "structure" because they still have some actual value that makes it to the codegen and everything.

To give you some perspective of why the test is even there, the original SDL was indentation-sensitive (was using indentation instead of `{ ... }`) so multi-line strings could potentially screw up the indentation. The current SDL is no longer sensitive, so the test is far more trivial now and just ends up testing the use of multi-line strings in a realistic context.

Reply:
Fixed here: 9392b823 but I'm in doubt that the test makes sense (i.e. tests anything new). As far as I remember I've disabled it to discuss later.

Reply:
It's OK even if it turns out to be redundant for this specific lexer and parser version. These syntax tests are fast and cheap enough. Also, this can come in handy when testing a parser re-write.